src/Counter.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+contract Counter {
```

Comment:
Move everything under `/src` to under the `/test` or `/contracts/mocks` as appropriate.
We'd like to maintain the `/contracts` folder as a collection of system contracts and reference-able base implementations and `/tests` can have the expansion of edge cases.
Similar to the OZ github